network/dag/notifier.go

```diff
@@ -291,7 +314,15 @@ func (p *notifier) Notify(event Event) {
 	}
 
 	if err := p.notifyNow(event); err != nil {
-		p.retry(event)
+		if isUnrecoverable(err) {
+			log.Logger().
+				WithError(err).
+				WithField(core.LogFieldTransactionRef, event.Hash.String()).
+				WithField(core.LogFieldEventSubscriber, p.name).
+				Errorf("Notify failed")
+		} else {
+			p.retry(event)
```

Comment:
So a hardcoded single retry and the error ignored again?

Reply:
retry() starts the retry mechanism and does not return anything